libs/vscode/copilot/src/lib/init-copilot.ts

```diff
@@ -68,7 +68,6 @@ const handler: ChatRequestHandler = async (
     return;
   }
   const workspacePath = getNxWorkspacePath();
-  const pmExec = (await getPackageManagerCommand(workspacePath)).dlx;
```

Comment:
removing this will prevent the direct invocation to work properly if `nx` is not installed globally

<img width="466" alt="Screenshot 2025-02-12 at 13 55 04" src="https://github.com/user-attachments/assets/84522a33-6534-4147-8cfa-b352ce6f9f0e" />